Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -3628,20 +3628,22 @@ DruidTestRTLCompiler >> gen_shortConditionalJumpFalse [
 		      allocateRegNotConflictingWith: live
 		      ifNone: [ ^ self unknownBytecode ].
 	live := live bitOr: (self registerMaskFor: t0).
-	(self ssValue: 0) moveToReg: t0.
+	(self ssValue: 0) copyToReg: t0.
 	self ssPop: 1.
 	self CmpCq: objectMemory falseObject R: t0.
 	jump1 := self JumpZero: 0.
 	self CmpCq: objectMemory trueObject R: t0.
 	jump2 := self JumpZero: 0.
 	self CallRT: ceSendMustBeBooleanTrampoline.
+	self annotateBytecode: self Label.
```

Comment:
What does it mean?

Reply:
This means that the mapping between machine code => bytecode